packages/radix-ui-themes/src/components/base-tab-list.props.ts

```diff
@@ -18,7 +18,7 @@ const baseTabListPropDefs = {
     type: 'enum',
     className: 'rt-r-jc',
     values: justifyValues,
-    default: 'start',
+    default: undefined,
```

Comment:
Can we just omit those instead of explicitly setting to `undefined`?

Reply:
I believe so, do we want to do that for all prop defs?

Reply:
If it works yeah I think so, unless there's a particular reason why we'd want that.

Reply:
I'll do it separately then in another PR on all prop defs to keep the history clean?